packages/material-ui/src/Badge/Badge.js

```diff
@@ -34,6 +34,7 @@ export const styles = theme => ({
     backgroundColor: theme.palette.color,
     color: theme.palette.textColor,
     zIndex: 1, // Render the badge on top of potential ripples.
+    transition: '225ms cubic-bezier(0.4, 0, 0.2, 1)',
```

Comment:
Can we use the transition theme helper?

Reply:
I didn't even realise the theme has a transition helper. I'll give this a go 👍